android-project/app/src/main/java/org/diasurgical/devilutionx/ImportActivity.java

```diff
@@ -20,6 +22,7 @@ public class ImportActivity extends Activity {
 
 	private static final int IMPORT_REQUEST_CODE = 0xD1AB70;
 
+	@RequiresApi(api = Build.VERSION_CODES.KITKAT)

```

Comment:
I didn't look into the entire code but having this annotation on the onCreate code will prevent the app to run on devices prior to this. Maybe the part that handles the file could be extracted to a new method and this annotation moved there.

Reply:
This is only accessible tought the shortcut list which is a feature that only exists in Android 25+, so preventing Android 18 from opening it isn't a problem.

Reply:
@AJenbo in that case, that's ok.